attributes/default.rb

```diff
@@ -18,4 +18,7 @@
 #
 
 default['build-essential']['compile_time'] = false
-default['build-essential']['msys']['path'] = "#{ENV['SYSTEMDRIVE']}\\msys"
+default['build-essential']['mingw']['prefix'] = "#{ENV['SYSTEMDRIVE']}\\mingw"
+
+# Change the defaults on pre-requisite cookbooks.
+default['seven_zip']['syspath'] = true
```

Comment:
The convention we've always used is only place defaults for the current cookbook in `attributes/default.rb`. Please move this back to `build-essential::_windows`.